ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml

```diff
@@ -590,6 +590,31 @@ tests:
     test:
     - chain: openshift-e2e-test-hypershift-qe-mgmt
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-hypershift-mce
+- as: aws-ipi-ovn-hypershift-mce-ibmz-guest-critical-f7
+  cron: 2 19 6,13,22,29 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      E2E_RUN_TAGS: '@critical'
+      MCE_VERSION: "2.5"
```

Comment:
The title states MCE 2.6, while the config is 2.5.  Is it expected ?

Reply:
Thanks!, update it, done